nautobot_netbox_importer/diffsync/netbox.py

```diff
@@ -0,0 +1,491 @@
+        default_reference={
+            "id": "Unknown",
+            "name": "Unknown",
+            "nestable": True,
+            "content_types": [],
+        },
+    )
+    adapter.configure_model(
+        "dcim.location",
+        fields={
+            "status": "status",
+            "location_type": _define_location_type,
+            "parent": _define_location,
+        },
+    )
+    adapter.configure_model(
+        "auth.user",
+        nautobot_content_type="users.user",
+        identifiers=["username"],
+        fields={
+            "last_login": None,
+            "is_superuser": None,  # NetBox 3.1 TBD verify
+            "password": None,  # NetBox 3.1 TBD verify
+            "user_permissions": None,  # NetBox 3.1 TBD verify
+        },
+    )
+    adapter.configure_model(
+        "auth.group",
+        identifiers=["name"],
+        fields={
+            "permissions": None,  # NetBox 3.1 TBD verify
+        },
+    )
+    adapter.configure_model(
+        "tenancy.tenant",
+        fields={
+            "group": "tenant_group",
+        },
+    )
+
+
+def _setup_circuits(adapter: SourceAdapter) -> None:
+    adapter.configure_model(
+        "circuits.circuit",
+        fields={
+            "type": "circuit_type",
+            "termination_a": "circuit_termination_a",
+            "termination_z": "circuit_termination_z",
+        },
+    )
+    adapter.configure_model(
+        "circuits.circuittermination",
+        fields={
+            "location": _define_location,
+        },
+    )
+
+
+def _setup_dcim(adapter: SourceAdapter) -> None:
+    adapter.configure_model(
+        "dcim.rackreservation",
+        fields={
+            "units": _define_units,
+        },
+    )
+    adapter.configure_model(
+        "dcim.rack",
+        fields={
+            "location": _define_location,
+            "role": _role_definition_factory(adapter, "dcim.rackrole"),
+        },
+    )
+    adapter.configure_model(
+        "dcim.cable",
+        fields={
+            "termination_a": "termination_a",
+            "termination_b": "termination_b",
+        },
+    )
+    adapter.configure_model(
+        "dcim.cabletermination_a",
+        extend_content_type="dcim.cable",
+        fields={
+            "termination_a": "termination_a",
+        },
+    )
+    adapter.configure_model(
+        "dcim.cabletermination_b",
+        extend_content_type="dcim.cable",
+        fields={
+            "termination_b": "termination_b",
+        },
+    )
+    adapter.configure_model(
+        "dcim.interface",
+        fields={
+            "status": "status",
+            "parent": None,  # TBD verify
+        },
+    )
+    manufacturer = adapter.configure_model(
+        "dcim.manufacturer",
+        default_reference={
+            "id": "Unknown",
+            "name": "Unknown",
+        },
+    )
+    adapter.configure_model(
+        "dcim.devicetype",
+        fields={
+            "front_image": None,  # TBD verify
```

Comment:
These exist on both sides, I am assuming since it is an image it's not part of the JSON blog. If that is the case it's okay to document and move on.

Reply:
Do not have data to test these for now, it's always empty in the available input. Will create follow-up ticket to cover this.

Reply:
May be easier to export a nautobot dataset that contains the front and back images to verify. I am 99% certain it's using django storages so they would not be in the database dump.

Reply:
Confirming, it’s represented in the NetBox export JSON data as follows: 
```
    "front_image": "devicetype-images/screenshot.png",
```